tests/model/utils/deletecontent.js

```diff
@@ -644,7 +644,16 @@ describe( 'DataController utils', () => {
 					{ rootName: 'bodyRoot' }
 				);
 
-				deleteContent( model, doc.selection );
+				model.change( writer => {
+					// Set selection to[<heading1>yyy</heading1>] in change() block due to selection post-fixer.
+					const range = new Range(
+						new Position( doc.getRoot( 'bodyRoot' ), [ 1 ] ),
+						new Position( doc.getRoot( 'bodyRoot' ), [ 2 ] )
+					);
+					writer.setSelection( range );
```

Comment:
You don't have to set that selection. `deleteContent()` can work with any selection.

Reply:
Also, it'd be good if you changed it in the `setData()` call above to something irrelevant to this test to not create a confusion.